controller/xt_default/impl.go

```diff
@@ -0,0 +1,47 @@
+package xt_default
+
+import (
+	"github.com/netfoundry/ziti-fabric/controller/xt"
+	"github.com/pkg/errors"
+)
+
+const (
+	Name = "single-endpoint"
```

Comment:
Should we use `default`? `single-endpoint` is kind of wordy.

Reply:
It has both "" and "default" as aliases, but I'm ok making "default" the name and "" an alias. We could also just use "single" as the name. Thoughts?

Reply:
Not sure I like the concept of "aliases". Wouldn't it be better to have a single, specific name for each strategy?

Reply:
It's there almost exclusively to support "" as an alias so we can have a strategy which handles existing services with no terminator strategy defined. Another approach would be write a migration which converts all services with a strategy of "" to "default" and also add code which assigns default to new services which don't define a strategy. Thoughts?

Reply:
I feel like it would be better to do the migration, rather than building the migration into the running system (which is what the alias does)?  ...especially if we're already having to migrate data. Why not just fix the Xt name, also?

Reply:
Removing aliases. 

I'm now also debating about whether we even want a strategy which only allows a single terminator. Even services which only have a single actual server running may want to bind via multiple routers to reduce single points of failure. I'm thinking about dropping xt_default entirely and making smartrouting the default. Does that line of reasoning seem reasonable?

Reply:
I mean... if "default" and `smartrouting` do the same thing in the presence of a single terminator, then there's no reason to have both? Dropping the "default" implementation makes sense to me. So, `smartrouting` would become the default?

Reply:
There's also nothing to limit the number of terminators that can be specified, even though "default" will only ever pick the first one... so, it would be pretty easy to get into a potentially weird state there. So, yes... it sounds like we should just make `smartrouting` be the default.

Reply:
:+1: will take that path